### ISSUE TITLE
html footer doesn't work with map panel

### ISSUE THREAD

**Issue by smrgeoinfo** [Esri/geoportal-server-catalog#110]
I need to show a footer at the bottom of our  Geoportal pages, and inserted a footer element after the tab-content div in the app.html template. Basic structure of the doc body is:
```
<div class="g-app">
    <nav class="navbar navbar-inverse" ...../>
    <div class="tab-content">
          <div id="searchPanel" ..../>
          <div id="mapPanel" ...../>
          <div id="aboutPanel" ...../>
    <footer class="page-footer">
</div>
```
The footer works fine with the searchPanel and aboutPanel, but displays just below the navbar when the mapPanel is selected. I'm guessing the iframe for the viewer/index.html page doesn't play well with the footer. Does anybody have a suggestion  on how to get the footer to work with the map panel?

page is currently at http://catalog-dev.iedadata.org:8080/geoportal/# but thats a dev site, so likely to change frequently.

**Comment by smrgeoinfo** [comment]
Interim solution-- I just put the footer element in the html templates for the searchPanel and aboutPanel, no footer in the mapPanel...

**Comment by KathleenPate** [comment]
I made the footer work by using CSS viewport height settings. Make the <body> tag almost 100% height of the viewport and then subtract the height of the footer. I found troubles with the Search Results panel floating down behind my original footer :-o This is what I came up with in the main.css stylesheet.  Modifying the viewport also allows the site to be viewed better from Tablets and some phones.

```
body {
  min-height: 99vmin;
}

.g-search-results {
  max-height: -webkit-calc(100vh - 19em);
  max-height:    -moz-calc(100vh - 19em); 
  max-height:         calc(100vh - 19em);
}
```
